test/Microsoft.Web.LibraryManager.Vsix.Test/Microsoft.Web.LibraryManager.Vsix.Test.csproj

```diff
@@ -14,11 +14,14 @@
     <ProjectReference Include="..\LibraryManager.Mocks\Microsoft.Web.LibraryManager.Mocks.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CSS.Core">
-      <HintPath>..\..\lib\Microsoft.CSS.Core.dll</HintPath>
+    <Reference Include="Microsoft.WebTools.Languages.Css">
```

Comment:
As above, make this `Include="..\..\lib\…`

Reply:
Updated.